versions.cfg

```diff
@@ -25,9 +25,12 @@ ZopeUndo = 4.1
 coverage = 4.2
 zdaemon = 4.1.0
 transaction = 1.4.4
-Sphinx = 1.3.6
+Sphinx = 1.5.1
+unittest2 = 1.1.0
```

Comment:
@jensens why is unittest2 reintroduced? I've been trying to remove it for a while